go/vt/servenv/status.go

```diff
@@ -301,7 +300,7 @@ func registerDebugBlockProfileRate() {
 		runtime.SetBlockProfileRate(rate)
 		log.Infof("Set block profile rate to: %d", rate)
 		w.Header().Set("Content-Type", "text/plain")
-		w.Write([]byte(message))
+		io.WriteString(w, message)
```

Comment:
Other places within here were using `io.WriteString`, so maintained the convention.